compiler_opt/es/inlining/inlining_worker.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Worker for inlining for size.
+"""
+
+import logging
+import os
+import concurrent.futures
+import tempfile
+import shutil
+import gin
+
+from compiler_opt.rl.inlining import inlining_runner
+from compiler_opt.rl import corpus
+from compiler_opt.distributed import worker
+from compiler_opt.rl import policy_saver
+from compiler_opt.es import policy_utils
+
+
+@gin.configurable
+class InliningWorker(worker.Worker):
+  """A worker that produces rewards for a given Inlining policy.
+
+  InliningWorker exposes a compile function, which
+  compiles a set of modules in parallel remotely, evaluates them with
+  llvm-size, and then computes the rewards based on the baseline size.
+  """
+
+  def _setup_base_policy(self):
```

Comment:
I think this is common from `RegallocTraceWorker`? Might be worth adding a TODO that these should share functionality.